examples/cp2k/preprunvasp.py

```diff
@@ -0,0 +1,97 @@
+from dflow import (
+    Workflow,
+    Step,
+    upload_artifact,
+    download_artifact,
+    InputArtifact,
+    OutputArtifact,
+    ShellOPTemplate,
+)
+from dflow.python import (
+    PythonOPTemplate,
+    OP,
+    OPIO,
+    OPIOSign,
+    Artifact,
+    TransientError,
+    FatalError,
+    BigParameter,
+    upload_packages,
+)
+from pathlib import Path
+import fpop
+from fpop.cp2k import PrepCp2k,Cp2kInputs,RunCp2k
+from fpop.preprun_fp import PrepRunFp
+from dflow.plugins.dispatcher import DispatcherExecutor
+
+prep_template_config={}
+prep_step_config={}
+run_template_config={}
+run_slice_config={}
+run_step_config={
+        "executor":{
+            "type": "dispatcher",
+            "machine_dict":{
+                "batch_type": "Bohrium",
+                "context_type": "Bohrium",
+                "remote_profile": {
+                    "input_data": {
+                        "job_type": "container",
+                        "platform": "ali",
+                        "scass_type": "c16_m32_cpu",
+                        }
+                    }
+                }
+            }
+    }
+prep_image_config={}
+run_image_config={
+    "command": "ulimit -s unlimited && ulimit -m unlimited && mpirun -n 16 --allow-run-as-root --oversubscribe cp2k.popt -i input.inp",
+    }
+upload_python_packages=list(fpop.__path__)
+
+steps = PrepRunFp(
+    "prep-run-vasp",
```

Comment:
Please change `vasp` to `cp2k`

Reply:
Thank you for your suggestion. I have changed the name.